### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by Sanne**
- https://hibernate.atlassian.net/browse/HSEARCH-2015
 - https://hibernate.atlassian.net/browse/HSEARCH-2016
 - https://hibernate.atlassian.net/browse/HSEARCH-2017
 - https://hibernate.atlassian.net/browse/HSEARCH-2018

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -143,16 +143,16 @@
 
         <!-- Dependency versions -->
         <slf4jVersion>1.6.4</slf4jVersion>
-        <luceneVersion>5.3.0</luceneVersion>
+        <luceneVersion>5.3.1</luceneVersion>
```
Comment (gunnarmorling) [inline pom.xml]: WFLY module update coming, too?

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks.

File: pom.xml (hunk 1 above)
Comment (Sanne) [inline pom.xml]: WDYM?

Comment (gunnarmorling) [inline pom.xml]: Whenever a PR is sent to WF to incorporate a HS version with this change,
that PR needs to update the WF Lucene modules, too. Just a reminder :)

Comment (Sanne) [inline pom.xml]: ok, sure.

**Pull request closed without merging**